x/pki/client/cli/query_noc_certificates.go

```diff
@@ -45,31 +50,51 @@ func CmdListNocIcaCertificates() *cobra.Command {
 	return cmd
 }
 
-func CmdShowNocIcaCertificates() *cobra.Command {
-	var vid int32
+func CmdShowNocCertificates() *cobra.Command {
+	var (
+		subject      string
+		subjectKeyID string
+	)
+
 	cmd := &cobra.Command{
-		Use:   "noc-x509-ica-certs",
-		Short: "Gets NOC ICA certificates (ICACs) by VID",
-		Args:  cobra.ExactArgs(0),
+		Use: "noc-x509-cert",
```

Comment:
It seems we have two very similar commands: `noc-x509-cert` and `noc-x509-certs`.
What if we combine them into one: `noc-x509-certs`, which can accept either VID/SKID or Subject/SKID.
We can even combine it with `all-noc-subject-x509-certs`, so we use `noc-x509-certs --subject` instead.

Reply:
Combined `noc-x509-cert` and `noc-x509-certs` commands into singe `noc-x509-cert`. 
Did not combine with `all-noc-subject-x509-certs` because it returns data in different representation.